src/EA.Weee.Web/Areas/Producer/Controllers/ProducerSubmissionController.cs

```diff
@@ -73,9 +74,15 @@ private async Task SetBreadcrumb(Guid organisationId, string activity)
 
         [HttpGet]
         [SmallProducerSubmissionContext]
-        public async Task<ActionResult> EditOrganisationDetails()
+        public async Task<ActionResult> EditOrganisationDetails(bool? redirectToCheckAnswers)
```

Comment:
If this is a nullable bool, would it not be helpful to give it a default null value? So nulls don't have to be explicitly passed in